molgenis-model-registry/src/main/frontend/src/store/getters.js

```diff
@@ -53,14 +53,12 @@ const mapNodeData = (entityTypes) => entityTypes.filter(entityType => (!entityTy
 
 const mapExtendedNodeData = (entityTypes) => entityTypes.filter(entityType => (entityType.extends && !entityType.isAbstract)).map(entityType => {
   const abstractEntityType = entityTypes.filter(aEntityType => (aEntityType.id === entityType.extends.id))[0]
```

Comment:
Instead of `.filter(predicate)[0]` you can also use [`.find(predicate)`](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Array/find)